azure-pipelines.yml

```diff
@@ -454,6 +454,36 @@ jobs:
       includePatterns:
         - "/hw/top_englishbreakfast/lowrisc_systems_chip_englishbreakfast_cw305_0.1.bit"
 
+- job: build_fpga_tests_cw310
```

Comment:
Are there advantages to a new job vs. packaging what comes out of Earl Grey Build & Test? Are we looking to keep the artifact size down?

Reply:
Currently, the Earl Grey Build & Test job does not build any of the FPGA artifacts (just the Verilator tests), and I didn't want to build anything unnecessary in the FPGA build/test path since it is currently the critical path for the CI. If we can get the disk cache working, then it would be great to build everything once at the beginning and then propagate the disk cache along, but at the moment, the disk cache hit rate is lower than expected for reasons I don't understand.